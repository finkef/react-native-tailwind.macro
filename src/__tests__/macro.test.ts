import pluginTester from 'babel-plugin-tester'
import plugin from 'babel-plugin-macros'
import path from 'path'

/* @ts-ignore */
import glob from 'glob-all'

pluginTester({
  plugin,
  pluginName: 'react-native-tailwind.macro',
  babelOptions: {
    configFile: path.join(__dirname, '../../macro.babel.config.js'),
  },
  snapshot: true,
  tests: glob
    .sync(['__fixtures__/**/*.js', '!__fixtures__/**/*.config.js'], {
      cwd: __dirname,
    })
    .map((file: string) => {
      return {
        title: path.basename(file),
        fixture: path.join(__dirname, file),
        snapshot: true,
      }
    }),
})
