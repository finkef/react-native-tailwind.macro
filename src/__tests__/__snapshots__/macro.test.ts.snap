// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`react-native-tailwind.macro single-prop.js: single-prop.js 1`] = `

import tw from '../../macro'

const Comp = () => <View tw="px-8" />

      ↓ ↓ ↓ ↓ ↓ ↓

const Comp = () => <View tw="px-8" />


`;
